clickhouse/columns/array.h

```diff
@@ -95,7 +95,7 @@ class ColumnArray : public Column {
 };
 
 template <typename ColumnType>
-class ColumnArrayT : public ColumnArray {
+class ColumnArrayT final : public ColumnArray {
```

Comment:
I assume that `final` is a leftover from a previous PR here. Otherwise, please explain why you need it.

Reply:
This is a cursoriness. Sorrry